lib/src/rewrite.rs

```diff
@@ -776,6 +779,7 @@ pub struct DuplicateCommitsStats {
 pub fn duplicate_commits(
     mut_repo: &mut MutableRepo,
     target_commits: &[CommitId],
+    mut target_descriptions: HashMap<CommitId, String>,
```

Comment:
nit: pass by reference? I don't think the extra `.clone()` cost would matter.

Reply:
Sure, I suppose.